accounts/api_keys.go

```diff
@@ -2,44 +2,62 @@ package accounts
 
 import (
 	"cards-against-humanity-api/models"
+	"fmt"
 	"github.com/pkg/errors"
 )
 
+const maxAPIKeyLimit = 3
+
 // DeactivateAPIKey soft deletes an API key for a user
 func (a *AccountClient) DeactivateAPIKey(userID int, keyID int) error {
-	// Soft delete API key
-	err := a.databaseClient.DeleteAPIKey(userID, keyID)
+	keyExists, err := a.databaseClient.DeleteAPIKey(userID, keyID)
 	if err != nil {
-		if err.Error() == "Not found" {
-			return ErrTokenNotFound
-		}
 		return errors.Wrap(err, "DeactivateAPIKey: Error deactivating API key")
 	}
+	if !keyExists {
```

Comment:
newline

Reply:
I usually don't leave a space between the error and checking for nil statements :) I think this is a fairly common convention in Go.